### ISSUE TITLE
Cannot find module './lexer'

### ISSUE THREAD

**Issue by jbpros** [joeattardi/json-colorizer#31]
Hi!

I'm unable to upgrade to 3.0.0. The following error occurs when trying to load json-colorizer:

```
[web] node:internal/modules/cjs/loader:1147
[web]   throw err;
[web]   ^
[web]
[web] Error: Cannot find module './lexer'
[web] Require stack:
[web] - /Users/jbpros/Projects/acw/node_modules/json-colorizer/dist/index.js
[web] - /Users/jbpros/Projects/acw/dist/main.js
[web]     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
[web]     at Module._load (node:internal/modules/cjs/loader:985:27)
[web]     at Module.require (node:internal/modules/cjs/loader:1235:19)
[web]     at require (node:internal/modules/helpers:176:18)
[web]     at Object.<anonymous> (/Users/jbpros/Projects/acw/node_modules/json-colorizer/dist/index.js:27:17)
[web]     at Module._compile (node:internal/modules/cjs/loader:1376:14)
[web]     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
[web]     at Module.load (node:internal/modules/cjs/loader:1207:32)
[web]     at Module._load (node:internal/modules/cjs/loader:1023:12)
[web]     at Module.require (node:internal/modules/cjs/loader:1235:19) {
[web]   code: 'MODULE_NOT_FOUND',
[web]   requireStack: [
[web]     '/Users/jbpros/Projects/acw/node_modules/json-colorizer/dist/index.js',
[web]     '/Users/jbpros/Projects/acw/dist/main.js'
[web]   ]
[web] }
```

Indeed, there is no `lexer.js` file in `dist/`, only `index.js`.

**Comment by joeattardi** [comment]
Oops, sorry, I goofed when publishing this version. I just released 3.0.1, give that a try. Thanks!

**Comment by jbpros** [comment]
Thanks, all good now! 🙇🏻
